src/pages/transactions/routes.tsx

```diff
@@ -52,15 +51,6 @@ export const transactionRoutes = (
         />
       }
     />
-    <Route
-      path=":id"
-      element={
-        <Guard
-          guards={[() => permission('view_transaction')]}
-          component={<Transaction />}
-        />
-      }
-    />
```

Comment:
I am not sure we wanna nuke whole page. @turbo124 do we want this? We are deleting page to view single transaction and making it visible only if we click on the table row (slideover).

Reply:
Here's minor work to return previous logic. But it was obvious to me not to duplicate places for the same data, so I removed the route and kept it only on the Slider.